### PR TITLE
Add support for additional mensas

### PR DESCRIPTION
**Pull request by fried-ice**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -408,9 +427,24 @@ def main():
     start_handler = CommandHandler('start', start)
     updater.dispatcher.add_handler(start_handler)
 
-    mensa_handler = CommandHandler('mensa', mensa)
+    mensa_handler = CommandHandler('mensa', alte_mensa)
```
Comment (MaxFleur) [inline src/main.py:431]: Now that the `alte_mensa` command is there, does it really make sense to still include the normal `mensa` command? Imo it's some sort of a duplicate, maybe even ambiguous.

File: src/main.py [hunk 2]
```diff
@@ -85,14 +85,33 @@ def mensa(update, context):
...
         img_url = elem["image"].lstrip("/") # For some reason, image URLs are prefixed with 2 leading slashes, but no protocol, remove them
-        print(img_url)
         # Do not send placeholder images
         if img_url.endswith("studentenwerk-dresden-lieber-mensen-gehen.jpg"):
             context.bot.send_message(chat_id=update.message.chat_id, text=markdown_highlight_char + elem["name"] + markdown_highlight_char, parse_mode="Markdown")
         else:
             context.bot.send_photo(chat_id=update.message.chat_id, photo=img_url, caption=markdown_highlight_char + elem["name"] + markdown_highlight_char, parse_mode="Markdown")
 
 
+def alte_mensa(update, context):
```
Comment (MaxFleur) [inline src/main.py:96]: It's not really clear where these numbers (4, 35 etc) come from. Maybe some more explanation (comments etc) or even a new variable for each number is more meaningful?

File: src/main.py (hunk 1 above)
Comment (fried-ice) [reply src/main.py:431]: The bot is still named AlteMensaBot.
It's only fair that we completely prefer the one and only Alte Mensa here IMHO.

File: src/main.py (hunk 2 above)
Comment (fried-ice) [reply src/main.py:96]: I added a comment right on top of the mensa function.

I'd prefer not to introduce new variables for this purpose. IMHO the process is pretty easily understandable as it is right now.
No need to add more complexity.

**Pull request merged**